lib/engines/content_block_manager/app/controllers/concerns/can_schedule_or_publish.rb

```diff
@@ -77,7 +77,7 @@ def scheduled_publication_date_params
   def review_update_url
     content_block_manager.content_block_manager_content_block_workflow_path(
       @content_block_edition,
-      step: ContentBlockManager::ContentBlock::Editions::WorkflowController::UPDATE_BLOCK_STEPS[:review_update],
+      step: :review,
```

Comment:
I'm getting really confused about what all the steps are and what they do, especially now they are in different files 😅  (though I don't think it's a bad idea to separate them) e.g. the difference between `review` and `review_update` and then also the difference between the `review` in show and the `review` in update. 
* could the names at least be unique between SHOW and UPDATE? 
I'm also wondering if all the actions could be in some kind of hash that would show their relationship/order 🤔 
* is the difference between review and review_update just the back link?

Reply:
I'll give it a go - would be good if we could combine `review` and `review_update` into one action, at the moment, it's a bit icky

Reply:
Maybe `review_new_block` and `review_updated_block` might be better naming?

Reply:
Or actually, if there is only one edition, we know it's a new block

Reply:
yeah those names are better I think, or yeah if it could be one action that would be even better.